iset.mm

```diff
@@ -63027,6 +63027,20 @@ would need to be strengthened (for example, to imply included middle) to
     ( wcel c1o cen wbr csn wceq en1eqsn ex ensn1g breq1 syl5ibrcom impbid ) ABC
     ZBDEFZBAGZHZOPRABIJOPRQDEFABKBQDELMN $.
 
+  ${
+    $d ph x y $.
+    $( A case where the antecedent of ~ snexg is not needed.  The class
+       ` { x | ph } ` is from ~ dcextest .  (Contributed by Mario Carneiro and
+       Jim Kingdon, 4-Jul-2022.) $)
```

Comment:
>  The class ` { x | ph } ` is from ~ dcextest .

What does it mean ?

Reply:
I'm not sure I have an answer for that.

For example, in the case of ` { x e. 1o | ph } ` I think our "official" name for it is "Mario's favorite ordinal". But what does it mean? I guess I sort of think of it as "a class which is in a superposition of (/) and 1o" but I'm not sure I'd use that kind of language in trying to describe it to someone else.

Reply:
What I mean is that I cannot make sense of the sentence "The class { x | ph } is from ~ dcextest ." in the comment. Maybe a coment could be:
> The singleton formed on the class ` { x | ph } ` where ` x ` does not occur in ` ph ` is a set.  This is in contrast to the general situation of ~  snexg .

(or for the second sentence, use your version).